apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -98,80 +162,81 @@ class WebLabTest extends Component {
-                the Web Lab tool
-              </label>
-              <div id="status-table">
-                <table>
-                  <tbody>
-                    <tr>
-                      <th>Check</th>
-                      <th>Status</th>
-                    </tr>
-                    <tr>
-                      <td>
-                        Connected to
-                        <code>https://downloads.computinginthecore.org</code>
-                      </td>
-                      <td className="computinginthecore">
-                        {this.renderTestStatus(STATUS_COMPUTING_IN_THE_CORE)}
-                      </td>
-                    </tr>
-                    <tr>
-                      <td>
-                        Connected to <code>https://codeprojects.org</code>
-                      </td>
-                      <td className="codeprojects">
-                        {this.renderTestStatus(STATUS_CODE_PROJECTS)}
-                      </td>
-                    </tr>
-                  </tbody>
-                </table>
-              </div>
-              <div>
-                <button
-                  type="submit"
-                  id="connect"
-                  className="btn btn-primary"
-                  onClick={this.runWebLabTest}
-                >
-                  Connect
-                </button>
-              </div>
-              {this.state.renderCallToAction && this.renderCallToAction()}
-              <br />
-            </div>
+      <div className={'contentContainer'}>
+        <div className={'content'} style={styles.content}>
+          <h1>Web Lab Network Support Check</h1>
+          <p>
+            Use this page to test your school's network support for our Web Lab
+            modules. Web Lab communicates with domains which may be blocked by
+            some school systems’ networks or device policies. If the network
+            check is unsuccessful, you will need to reach out to your school's
+            IT department to update your school's firewall settings or device
+            policies.
+          </p>
+          <p>
+            When checking network support, try to meet as many of these criteria
+            as possible:
+          </p>
+          <li>Use your school's internet connection</li>
+          <li>Use a student computer</li>
+          <li>Log into the computer with a student account</li>
+          <br />
+          <p>
+            Note: You do not need to be logged in to Code.org for this test to
+            be successful.
+          </p>
+          <label htmlFor="connect">
+            Click "Connect" to check whether your school's network supports the
+            Web Lab tool
+          </label>
+          <button
+            type="submit"
+            id="connect"
+            className="btn btn-primary"
+            onClick={this.runWebLabTest}
+          >
+            Connect
+          </button>
```

Comment:
Does anything odd happen if the user presses the button rapidly?

Either way, should we disable this button while the test is running?

We could also consider leaving it permanently disabled, or even removing it, once the test has been run once?  Or do we want to allow re-runs?

Reply:
Here's a video of the test running when the button is rapidly pressed. You can't see all the clicks but I'm clicking about as quickly as I can in the video! The new button disabling logic appears to keep anything odd from happening. Before I was properly disabling the button, I could cause the Bramble test to fail intermittently if I ran it rapidly over and over again.

https://user-images.githubusercontent.com/26844240/214958929-6d48f8aa-9839-401c-9572-9ff080a80830.mov

Reply:
Looks good.  Long ago I worked on some UI that had a lot of bugs that we would discover by doing rapid game controller mashing :)